src/pages/design/guidelines.js

```diff
@@ -1,19 +1,45 @@
-import React from 'react'
-import Layout from '../../components/Layout'
-import GuidelineHeader from '../../components/guideline-header/guideline-header'
+import React from 'react';
+import Layout from '../../components/Layout';
+
+import GuidelineHeader from '../../components/guideline-header/guideline-header';
 import GuidelineLeftNav from '../../components/guideline-left-nav/guideline-left-nav';
-import './guidelines.scss'
+
+import designImg from './../../img/design.png';
 
 export default class DesignGuidelineIndexPage extends React.Component {
   render() {
+    const { location } = this.props;
     return (
-      <Layout>
-        <GuidelineHeader></GuidelineHeader>
-        <GuidelineLeftNav></GuidelineLeftNav>
-        <section className="section design-guidelines">
-          <div className="welcome"></div>
-        </section>
+      <Layout location={location}>
+        <GuidelineHeader />
+        <div
+          css={{
+            display: 'flex',
+            flexDirection: 'row',
+            alignItems: 'start',
+            width: '100%'
+          }}
+        >
+          <GuidelineLeftNav />
+          <div
+            css={{
+              width: '100%'
+            }}
+          >
+            <div
+              css={{
+                width: '100%',
+                height: '200px',
+                backgroundColor: '#f8f9fb',
+                height: '682px',
+                backgroundImage: 'url(' + designImg + ')',
```

Comment:
we need to look into using `gatsby-image` to load background images. Doing like this will not optimise the images.